rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -122,6 +123,12 @@
 
     public Coordinator() {
         lraService = LRARecoveryModule.getService();
+        recoveryCoordinator = new RecoveryCoordinator();
+    }
+
+    @Path(RECOVERY_COORDINATOR_PATH_NAME)
+    public RecoveryCoordinator getRecoveryCoordinator() {
+        return recoveryCoordinator;
```

Comment:
@ochaloup correctly pointed out that this will use the same instance of RecoveryCoordinator for each call. I believe that quarkus defaults to application scoped beans for JAX-RS resources so I don't think that this should be an issue, just something to keep in mind.

Reply:
It's irrelevant because RecoveryCoordinator is not a singleton, ie you can create however many instances you like, ie the front end API is stateless.